scripts/index.js

```diff
@@ -0,0 +1,117 @@
+console.log("fingers crossed it's sunny")
+
+let temperatureValue = 70;
+
+// sky block functions
+
+const updateSky = () => {
+    const inputSky = document.getElementById("sky-options").value;
+    const skyContainer = document.getElementById("sky-art");
+    let skyArt = "🔆☀️🌞🌤️⛅🌥️🌦️";
+    if (inputSky == "sunny") {
+        skyArt = "☁️ ☁️ ☁️ ☀️ ☁️ ☁️";
+    } else if (inputSky == "cloudy") {
+        skyArt = "☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️";
+    } else if (inputSky == "rainy") {
+        skyArt = "🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧";
+    } else if (inputSky == "snowy") {
+        skyArt = "🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨";
+    } else {
+        skyArt = "🔆☀️🌞";
+    }
+    skyContainer.textContent = skyArt;
+};
+
+
+const updateGarden = (currentTemperature) => {
+    const landArtContainer = document.getElementById("land-art");
+    let landArt = "🌲🌲🌲";
+    if (currentTemperature >= 80) {
+        landArt = "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂";
+    } else if (currentTemperature >= 70) {
+        landArt = "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷";
+    } else if (currentTemperature >= 60) {
+        landArt = "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃";
+    } else if (currentTemperature >= 50) {
+        landArt = "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲"
+    } else {
+        landArt = "🌲";
+    }
+    landArtContainer.textContent = landArt;
+};
+
+// temperature block functions
+
+const updateTemperatureColors = (currentTemperature) => {
+    const temperatureValueContainer = document.getElementById("temperature-value");
+    let color = "black";
```

Comment:
Is this black color ever used? If not we can simply declare color with `let color`